db/migrate/20230828145454_add_migrated_on_fields.rb

```diff
@@ -0,0 +1,6 @@
+class AddMigratedOnFields < ActiveRecord::Migration[7.0]
+  def change
+    add_column :lettings_logs, :values_updated_at, :datetime
```

Comment:
if this is just a migrated at date, why don't we call it that?

Reply:
Cause we decided to make this more generic, so that if in the future we need to adjust values we can use the same column. Added some info in description